airflow-core/newsfragments/48419.significant.rst

```diff
@@ -0,0 +1,18 @@
+Unused attribute ``num_times_parse_dags`` of SchedulerJobRunner has been removed.
+
+This change removes an unused attribute from the SchedulerJobRunner class to clean up the codebase.
+
+* Types of change
+
+  * [ ] Dag changes
+  * [ ] Config changes
+  * [ ] API changes
+  * [ ] CLI changes
+  * [ ] Behaviour changes
+  * [ ] Plugin changes
+  * [ ] Dependency changes
+  * [x] Code interface changes
+
+* Migration rules needed
+.. e.g.,
+.. * Attribute ``num_times_parse_dags`` removed from ``airflow.jobs.scheduler_job.SchedulerJobRunner``
```

Comment:
Not entirely sure that we need a migration rule, as I highly doubt that someone actually imports the SchedulerJobRunner directly (unless they actually customize Airflow). Kept it commented out for now.

Reply:
Yeah, we don't need a migration rule here, this is not part of the "public" interface in any shape or form